r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -415,3 +406,160 @@ test_that("mutate and write_dataset", {
       summarize(mean = mean(integer))
   )
 })
+
+# PACHA ADDITIONS ----
+# READ THIS CAREFULLY PLEASE, IT'S MY 1ST DAY WRITING THIS KIND OF SENSITIVE TESTS
+
+# similar to https://github.com/tidyverse/dplyr/blob/master/tests/testthat/test-mutate.r#L1-L10
+# the rest of that test belongs in L55-62 here
+test_that("empty mutate returns input", {
+  # dbl2 = 5, so I'm grouping by a constant
+  gtbl <- group_by(tbl, dbl2)
+
+  expect_dplyr_equal(input %>% mutate(!!!list()) %>% collect(), tbl)
+
```

Comment:
Add a test equivalent to the one you deleted at L55-62:
```suggestion
  expect_dplyr_equal(input %>% mutate() %>% collect(), tbl)
  expect_dplyr_equal(input %>% mutate(!!!list()) %>% collect(), tbl)
```